### DISCUSSION TITLE
Shortcut command that globalsearches when I copy a RITM/INC/IMS

### DISCUSSION THREAD

**Discussion by Alejandrohe08** [arnoudkooi/SN-Utils#292] [Q&A]
Hello, as I mentioned on the title I'm trying to make a shortcut command that can globally search (using the search command) whatever I copy or click+randomkey, it would be useful for my team since we are limited right now and it would save us time from going to agent workspace to the prod one, is there any way to make this happen? Thanks!

**Comment by arnoudkooi** [comment]
Did you try the built in` /search INC001010`? it tries the exact match first...

**Reply by Alejandrohe08** [reply]
I did try it and it works perfectly but I'm trying to skip opening the console and pasting the ticket number, so I can search for it directly. In my head it would work like opening it in a new tab (using the scroll wheel button) but I'm not sure if this is too much to ask for 🤔

**Reply by arnoudkooi** [reply]
I don't have a direct solution for that. Not the global search is accessible via the context menu.
On mac you can hold control to overwrite a ServiceNow context menu.

<img width="985" alt="image" src="https://user-images.githubusercontent.com/840781/186665469-6b14701a-2617-4e0d-9661-6c9a30ba90b6.png">

**Reply by Alejandrohe08** [reply]
Thanks for the reply Arnoud! This actually worked very well; my final question would be: **Is there a way to assign that "Search /GoTo" tool to the shortcut slash command?** That way I think it would be pretty much my original idea because I would select the text, then press one of my mouse macros (configured to be the same hotkey as the shortcut slashcommand) and the ticket would open in a new tab only by pressing two buttons. Thanks for helping me out!
